docs/components/query.md

```diff
@@ -136,6 +136,14 @@ Querier also allows to configure different timeouts:
 
 If you prefer availability over accuracy you can set tighter timeout to underlying StoreAPI than overall query timeout. If partial response strategy is NOT `abort`, this will "ignore" slower StoreAPIs producing just warning with 200 status code response.
 
+### Timing differences for metric scraping
+
+For example: If 2 prom replicas are scraping the same metrics (prom A and prom B) and if A is consistently reading metrics traffic every 15s at the 5 (every minute at 5, 20, 35, 50) and B is consistently reading the same metrics every 15s at the 14 (every minute at 14, 29, 44, 59) what is the impact to the total metric over a minute if the A process stops? Or if B stops?
```

Comment:
```suggestion
For example: If 2 Prometheus replicas are scraping the same metrics (Prometheus A and Prometheus B) and if A is consistently reading metrics traffic every 15s starting at the 5 mark (every minute at 5, 20, 35, 50) and B is consistently reading the same metrics every 15s at the 14 mark (every minute at 14, 29, 44, 59) what is the impact to the total metric over a minute if the A process stops? Or if B stops?
```

Reply:
Also please check the DCO here: https://github.com/thanos-io/thanos/pull/5004/checks?check_run_id=4639494785 Thanks!

Reply:
Made the suggested changes and created a new pull request https://github.com/thanos-io/thanos/pull/5057  Please provide feedback.